app/js/demo-controller.js

```diff
@@ -152,6 +191,12 @@ export class DemoPaywallController {
       setTimeout(() => {
         response.complete().then(() => {
           log('subscription has been confirmed');
+          this.subscriptions.getPropensityModule().then(module => {
+            const jsonResponse = response && response.json();
+            const entitlementsJson = jsonResponse && jsonResponse['entitlements'];
+            const products = this.getProductList_(entitlementsJson);
+            module.sendEvent('payment_complete', {'product': products});
```

Comment:
nit: can we make this 'products': products?

Reply:
I am following the instructions a developer would from the API doc:
"
  /**
   * ACTION_PAYMENT_COMPLETED.
   * User has made the payment for a subscription.
   * The JSON block can provide the product user paid for.
   * For example; {'product': 'basic-monthly'}
   */
  ACTION_PAYMENT_COMPLETED: 'payment_complete',
"

I actually expect this to be one product only. But using products as a variable to notify a possible list of products found in entitlements response.